pkg/services/sqlstore/migrations/accesscontrol/test/ac_test.go

```diff
@@ -153,7 +153,8 @@ func TestMigrations(t *testing.T) {
 		{
 			desc: "with editors can admin",
 			config: &setting.Cfg{
-				EditorsCanAdmin:        true,
+				EditorsCanAdmin: true,
+				// nolint:staticcheck
```

Comment:
@gamab -- is this still valid?  should it be removed now?

https://github.com/grafana/grafana/pull/44065